akka-serialization-jackson/src/main/scala/akka/serialization/jackson/AkkaSerializationModule.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.serialization.jackson
+
+import com.fasterxml.jackson.core.{ JsonGenerator, JsonParser, ObjectCodec }
+import com.fasterxml.jackson.databind.{ DeserializationContext, JsonNode, SerializerProvider }
+import com.fasterxml.jackson.databind.deser.std.StdScalarDeserializer
+import com.fasterxml.jackson.databind.ser.std.StdScalarSerializer
+import akka.serialization.{ SerializationExtension, Serializer, Serializers }
+
+final class AkkaSerializationSerializer extends StdScalarSerializer[AnyRef](classOf[AnyRef]) with ActorSystemAccess {
+  private val serialization = SerializationExtension(currentSystem())
```

Comment:
can we be sure that the `AkkaSerializationSerializer` is instantiated at the right point (inside other serialization), or should this be `lazy`. Will it cache the `AkkaSerializationSerializer` instances and then it might not be correct system. Maybe best to make it a `def`?

Reply:
👍